packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -8,10 +8,15 @@ import geoLocationDetails from '../../demo/get-geo-location.json';
 import customer from '../../demo/get-customer.json';
 import storageMock from '../../../test-utils/local-storage/local-storage-mock';
 import addressService from '../../services/addressService';
+import basketApi from '../../services/basketApi';
+import checkoutApi from '../../services/checkoutApi';
+import addressGeocodingApi from '../../services/addressGeocodingApi';
+import orderPlacementApi from '../../services/orderPlacementApi';
+import accountApi from '../../services/accountApi';
 import {
     mockAuthToken, mockAuthTokenNoNumbers, mockAuthTokenNoMobileNumber
 } from '../../components/_tests/helpers/setup';
-import { version as applicationVerion } from '../../../package.json';
+// import { version as applicationVerion } from '../../../package.json';
```

Comment:
Commented code

Reply:
Given the need for the contract tests, lets get this merged and fix this in a separate ticket 👍🏼 

Good spot though!